ui/lib/pki/addon/components/page/pki-tidy-status.ts

```diff
@@ -57,24 +60,26 @@ export default class PkiTidyStatusComponent extends Component<Args> {
   tidyStatusGeneralFields = [
     'time_started',
     'time_finished',
-    'last_auto_tidy_finished',
     'cert_store_deleted_count',
+    'revoked_cert_deleted_count',
     'missing_issuer_cert_count',
-    'revocation_queue_deleted_count',
   ];
 
   tidyStatusConfigFields = [
     'tidy_cert_store',
-    'tidy_revocation_queue',
```

Comment:
duplicate (enterprise only, below) - this should have originally been `tidy_revoked_certs`